about-us.html

```diff
@@ -180,7 +180,7 @@
         </div>
 
         <!--   Most styles in Global.css but also in pages/about-us/index.css  -->
-        <section class="content">
+        <section class="content white">
```

Comment:
Doing this because it had a transparent background and so was inheriting the app's background color (which is now blue)